float32 time
float32 body_angle
float32 motor_angleA
float32 motor_angleB
float32 leg_angleA
float32 leg_angleB
float32 body_ang_vel
float32 motor_velocityA
float32 motor_velocityB
float32 leg_velocityA
float32 leg_velocityB
float32 xPosition
float32 yPosition
float32 zPosition
float32 xVelocity
float32 yVelocity
float32 zVelocity
float32 horizontal_velocity
float32 motor_currentA
float32 motor_currentB
uint8 toe_switch
uint8 command
float32 motor_torqueA
float32 motor_torqueB
